mir-module/src/main/java/org/mycore/mir/editor/MIREditorTools.java

```diff
@@ -0,0 +1,24 @@
+package org.mycore.mir.editor;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Entities;
+import org.w3c.dom.Node;
+
+public class MIREditorTools {
```

Comment:
Codacy found an issue: [The utility class name 'MIREditorTools' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5834770)